tasks/mysql_users.yml

```diff
@@ -1,4 +1,47 @@
 ---
+
+- name: "Check if {{ mariadb_sst_username }} exists in the system"
```

Comment:
I think that it would be better to place this to check task file.

Reply:
I thought that too... but at `check` time the packages are not installed yet and the probability of a missing mysql user is higher.